cfg/cfg.go

```diff
@@ -188,6 +197,9 @@ func Print() {
 	if SelfSignedCertificateFilePath != "" {
 		log.Printf("  Self signed certificate %s", SelfSignedCertificateFilePath)
 	}
+	if CABundleDirPath != "" {
```

Comment:
Having `SelfSignedCertificateFilePath` and `CABundleDirPath` at the same time is a bit confusing... Could we just go with one - CABundleDir?

They have slightly different documentation but in fact the only difference according to
```
// ConfigureCertPool trusts given certificates
// CADirPath is a path to directory with CA certificates files
func ConfigureCertPool(CAFilePath string, CADirPath string) {
```
their type - file and dir... And it's possible to specify both.

Reply:
I am not sure we should. CAFile is the file which contains Che server self-signed certificate, where CABundle is a set of all trusted CA certificates inside Openshift cluster (however it doesn't has the cert from CAFile - it is not clear to me why, but that's how Openshift works). I agree that the comments might be confusing, thought. I'll update them.